vts/trustedservices/trustedservices_grpc.go

```diff
@@ -360,38 +360,44 @@ func (o *GRPC) GetAttestation(
 		return o.finalize(appraisal, err)
 	}
 
-	appraisal.EvidenceContext.ReferenceId = extracted.ReferenceID
+	appraisal.EvidenceContext.ReferenceIds = extracted.ReferenceIDs
 
 	o.logger.Debugw("constructed evidence context",
-		"software-id", appraisal.EvidenceContext.ReferenceId,
-		"trust-anchor-id", appraisal.EvidenceContext.TrustAnchorId)
+		"software-id", appraisal.EvidenceContext.ReferenceIds,
+		"trust-anchor-id", appraisal.EvidenceContext.TrustAnchorIds)
 
-	endorsements, err := o.EnStore.Get(appraisal.EvidenceContext.ReferenceId)
-	if err != nil && !errors.Is(err, kvstore.ErrKeyNotFound) {
-		return o.finalize(appraisal, err)
-	}
+	var tendorsements []string
+	for _, reference := range appraisal.EvidenceContext.ReferenceIds {
```

Comment:
Maybe, since it's an actual identifier, instead of `reference` call this `refvalID`

Reply:
agree ! i have changed it!